bundles/org.openhab.binding.renault/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -69,26 +78,88 @@
 				<description>Interval the car is polled in minutes.</description>
 				<default>10</default>
 			</parameter>
+			<parameter name="updateDelay" type="integer" unit="s" min="5" max="120">
+				<label>Update Delay</label>
+				<description>How long to wait for commands to reach car and update to server in seconds.</description>
+				<default>30</default>
+			</parameter>
 		</config-description>
 	</thing-type>
 
-	<!-- Sample Channel Type -->
-	<channel-type id="hvacstatus">
-		<item-type>Switch</item-type>
-		<label>HVAC Status</label>
+	<channel-type id="batteryavailableenergy">
+		<item-type>Number:Energy</item-type>
+		<label>Battery Energy Available</label>
+		<state pattern="%.1f kWh" readOnly="true"></state>
+	</channel-type>
+	<channel-type id="plugstatus">
+		<item-type>String</item-type>
+		<label>Plug Status</label>
+		<description>Status of charging plug.</description>
 		<state readOnly="true"></state>
 	</channel-type>
-	<channel-type id="image">
+	<channel-type id="chargingstatus">
 		<item-type>String</item-type>
-		<label>Image URL</label>
-		<description>Image URL of MyRenault</description>
+		<label>Charging Status</label>
 		<state readOnly="true"></state>
 	</channel-type>
+	<channel-type id="chargingmode">
+		<item-type>String</item-type>
+		<label>Charging Mode</label>
+	</channel-type>
+	<channel-type id="chargingremainingtime">
+		<item-type>Number:Time</item-type>
+		<label>Charging Time Remaining</label>
+		<category>Time</category>
+		<state pattern="%d %unit%" readOnly="true"></state>
+	</channel-type>
+	<channel-type id="estimatedrange">
+		<item-type>Number:Length</item-type>
+		<label>Estimated Range</label>
+		<description>Estimated range of the car.</description>
+		<state pattern="%.1f %unit%" readOnly="true"></state>
+	</channel-type>
 	<channel-type id="odometer">
 		<item-type>Number:Length</item-type>
 		<label>Odometer</label>
 		<description>Total distance travelled</description>
 		<state pattern="%.1f %unit%" readOnly="true"></state>
 	</channel-type>
+	<channel-type id="hvacstatus">
+		<item-type>String</item-type>
+		<label>HVAC Status (ON | OFF | PENDING)</label>
+		<state readOnly="false">
+			<options>
+				<option value="ON">ON</option>
+				<option value="PENDING">PENDING</option>
+				<option value="OFF">OFF</option>
```

Comment:
Do you really want labels fully in uppercase ?

Reply:
Yes because then the switch (ON OFF) habpannel widgets work...

Reply:
You tell me that you are labelling your options as a workaround for habpannel ?! I hope this is not the case. Note that these labels will be translated with different labels in other languages.

Reply:
Ok I think I have lowercase labels and uppercase values and enums for the options.  I think this is correct now but will need to test it.